gammapy/spectrum/core.py

```diff
@@ -241,6 +241,33 @@ def spectral_index(self, energy, dz=1e-3):
 
         return spectral_index
 
+    def rebin(self, parameter):
```

Comment:
Minor Comment:
The equivalent method on `SkyImage` is called `downsample()` (see [here](https://github.com/gammapy/gammapy/blob/master/gammapy/image/core.py#L563)), just in case we want to go for a more uniform API between our data structures...